### ISSUE TITLE
Bug: `paged` does not pass `kwargs` to operation

### ISSUE THREAD

**Issue by jrieke** [fastai/ghapi#24]
On [this line](https://github.com/fastai/ghapi/blob/master/ghapi/page.py#L16), the `kwargs` that `paged` receives are not passed on to the operation (as is described in the docs [here](https://ghapi.fast.ai/page.html#paged) and the docstring in the line above). 

This produces an error for me when using paged with a custom header, e.g.: 

```python
paged(api.activity.list_stargazers_for_repo, "fastai", "ghapi", headers={"Accept": "application/vnd.github.v3.star+json"})
```

**Comment by jrieke** [comment]
Made a small PR to fix this, see #25

**Comment by hamelsmu** [comment]
Thanks!
